test/spec/ol/tilegrid/tilegrid.test.js

```diff
@@ -404,16 +404,17 @@ describe('ol.tilegrid.TileGrid', function() {
     it('respects configuration options', function() {
       const tileGrid = createXYZ({
         extent: [10, 20, 30, 40],
+        maxResolution: 10 / 128,
         minZoom: 1,
         maxZoom: 2,
         tileSize: 128
       });
-      expect(tileGrid.getExtent()).to.eql([10, 20, 30, 40]);
```

Comment:
Why was the extent assertion removed?

Reply:
Seems I overwrote it instead of adding new line

Reply:
The handlebars error is getting rather repetitive!